js/skin.js

```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
    * @public
    * @description the function is called when we stop the rotation
    */
-  handleVrPlayerMouseUp: function() {
+  handleVrPlayerMouseUp: function(e) {
```

Comment:
Can you add the new argument in the jsdoc?

Reply:
\+